libs/perun/services/src/lib/store.service.ts

```diff
@@ -61,7 +66,21 @@ export class StoreService {
   get(...keys: string[]) : any {
     let currentValue: string;
 
-    if (this.instanceConfig !== undefined) {
+    if (this.branding !== '') {
+      const brandingConfig = this.instanceConfig['brandings'][this.branding];
+      for (let i = 0; i < keys.length; ++i) {
+        if (i === 0) {
+          currentValue = brandingConfig[keys[i]];
+        } else {
+          if (currentValue === undefined) {
+            break;
+          }
+          currentValue = currentValue[keys[i]];
+        }
+      }
+    }
+
+    if (this.instanceConfig !== undefined && currentValue === undefined) {
       for (let i = 0; i < keys.length; ++i) {
         if (i === 0) {
           currentValue = this.instanceConfig[keys[i]];
```

Comment:
A couple of lines below is an error printed to the console if the configuration for the branding is not found. Could you please change it, so this error is not printed whilst search the `brandings` property? 
```
    console.error('Missing value in default config: ' + keys);
```